portal/src/texts/blog/iePlussJokulErlikSant/iePlussJokulErlikSant.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: IE + Jøkul = Sant
+description: Hvordan kan du bruke Jøkul i IE11
+author: Jo Emil Holen
+publishDate: "2020.08.05"
+---
+
+"Vi må støtte IE 11" trenger ikke være den verste setningen en utvikler kan høre. Med noen enkle steg kan man komme veldig langt uten å hverken måtte snu opp-ned på markup eller kode.
+
+## Steg én: Polyfill JavaScript for IE 11
+
+Legg til følgende konfigurasjon i `package.json`:
+
+```json
+"browserslist": {
```

Comment:
Hvorfor ikke bruke browserlisten til fremtind?

```
"browserslist": {
      "extends @fremtind/browserslist-config-jkl",
      "ie 11"
}
```

Reply:
Godt poeng! Sånn jeg leser den configen trenger man ikke spesifisere "ie 11" heller, iom. at den blir (enn så lenge) tatt med av `>0.2%` og `ie 11`.

Reply:
Ja, men for blogpostens skyld så ville jeg tatt med ie11, hvis din app krever IE støtte så bør man spesifisere det. På sikt vil vi nok fjerne ie 11 fra vår default browserlist :)

Reply:
Og bruk av prosentene er litt ustabilt, så sånn jeg ser det burde man unngå å bruke de. Fra en release til en annen, så kan bundlen din endre seg en god del bare fordi statestikken din har endret seg litt.

Reply:
e9! 😄